docs/content/users/extend/customization-extendibility.md

```diff
@@ -13,20 +13,27 @@ You can set custom environment variables in several places:
 
 1. An optional, project-level `.ddev/.env` file provides environment variables to all DDEV containers, including any additional services or add-ons. It can look something like this:
```

Comment:
This statement is not true for DDEV v1.23.4, because `.ddev/.env` adds variables inside containers only for the `web` container.

But it will become true after this PR goes in.